src/System.Net.Primitives/tests/FunctionalTests/project.json

```diff
@@ -2,7 +2,8 @@
   "dependencies": {
     "System.Diagnostics.Contracts": "4.0.0",
     "System.Runtime": "4.0.20",
-    "System.Threading.Tasks": "4.0.10",
+    "System.Threading.Tasks": "4.0.11-rc2-23610",
```

Comment:
These need to remain generic versions (please remove the  `-rc2-23610` part) to ensure we're testing against the latest packages.
Also, any reason (newly required public API) you are changing them to 4.0.11?